crates/testing/tests/vid_task.rs

```diff
@@ -83,11 +83,14 @@ async fn test_vid_task() {
         1,
     );
 
-    let vote_token = vid_exchange
-        .make_vote_token(ViewNumber::new(2))
-        .unwrap()
-        .unwrap();
-    let vid_vote = vid_exchange.create_vid_message(block.commit(), ViewNumber::new(2), vote_token);
+    let vid_vote = VIDVote2::create_signed_vote(
```

Comment:
So nice that the vote token + message creation are now combined into one function. 👍